packages/sit-onyx/src/components/OnyxColorSchemeDialog/OnyxColorSchemeDialog.vue

```diff
@@ -0,0 +1,166 @@
+<script lang="ts" setup>
+import { computed, ref, watchEffect } from "vue";
+import { injectI18n } from "../../i18n";
+import OnyxButton from "../OnyxButton/OnyxButton.vue";
+import OnyxDialog from "../OnyxDialog/OnyxDialog.vue";
+import OnyxHeadline from "../OnyxHeadline/OnyxHeadline.vue";
+import OnyxRadioButton from "../OnyxRadioButton/OnyxRadioButton.vue";
+import autoImage from "./auto.svg?raw";
+import darkImage from "./dark.svg?raw";
+import lightImage from "./light.svg?raw";
+import type { ColorSchemeValue, OnyxColorSchemeDialogProps } from "./types";
+
+const props = withDefaults(defineProps<OnyxColorSchemeDialogProps>(), {
+  open: false,
+});
+
+const emit = defineEmits<{
+  /**
+   * Emitted when the color scheme should be changed.
+   */
+  "update:modelValue": [value: ColorSchemeValue];
+  /**
+   * Emitted when the dialog should be closed.
+   */
+  close: [];
+}>();
+
+const currentValue = ref(props.modelValue);
+watchEffect(() => (currentValue.value = props.modelValue));
+
+const { t } = injectI18n();
+
+const options = computed<
+  { value: ColorSchemeValue; description: string; image: string; label: string }[]
```

Comment:
should the type definition also be moved to the `packages/sit-onyx/src/components/OnyxColorSchemeDialog/types.ts` like we do on many other components like checkbox/radio ? or not in this case because the devs don't need to import that type

Reply:
Thats only a local type so I don't want to export it :)